font-types/src/path.rs

```diff
@@ -0,0 +1,24 @@
+/// Interface for accepting a sequence of path commands.
+///
+/// This is a general abstraction to unify ouput for processes that decode and/or
+/// transform outlines.
+///
+/// The type parameter `T` represents the coordinate type.
+pub trait PathSink<T> {
```

Comment:
noob question: is Into<T> a bad idea here and if so why?

Reply:
Do you mean `T` as the coordinate type, or do you mean `Into<T>` as an alternative to this whole interface?

Reply:
Also: is there a strong argument for having distinct coordinate types? How all do we expect this to be used?

Reply:
I'm honestly not sure. This was initially non-generic and used f32. When I was copying it over to read-fonts, I attempted to make both PathSink and to_path generic, but ended up stalling on the necessary division by 2 for computing mid points. I left PathSink as a generic because it didn't seem to hurt. Happy to go back to unconditional f32s if this seems like premature abstraction.

I also need more context for `Into<T>`

Reply:
Yea I'd vote for just leaving it as f32 🤷

Reply:
Seems reasonable. I'll change it back.